sklearn/tests/test_metaestimators_metadata_routing.py

```diff
@@ -277,7 +284,10 @@ def enable_slep006():
     LassoLarsCV(),
     MultiTaskElasticNetCV(),
     MultiTaskLassoCV(),
+    OneVsOneClassifier(ConsumingClassifier()),
+    OneVsRestClassifier(ConsumingClassifier()),
     OrthogonalMatchingPursuitCV(),
```

Comment:
I think we might need to remove OrthogonalMatchingPursuitCV from here.

Reply:
Oh yes, they don't belong there,  (they all got back in while auto-merging). I really need to remind myself to check that updating the branch did what I thought it did.